src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/EditIssuePage.kt

```diff
@@ -36,9 +37,21 @@ class EditIssuePage(
                 .overwriteIfPresent("description")
         }
         form.fillRequiredFields()
+        selectResolution()
         return this
     }
 
+    private fun selectResolution() {
```

Comment:
You're mixing 3 different features in the same PR. It means that if there's a problem with any of them, all get blocked. It makes merging and reviewing harder. I recommend splitting PRs per feature.

Reply:
My mistake, I meant to just push the changes so everyone in the team can view/edit it. Did not realise that I raised a PR earlier.

Reply:
You can create a new branch for the purposes of getting XL results. Pushing many independent changes via a single branch won't work.

Do you want to merge these changes? If so, split them. If not, then let's close this PR, because reviewers are spending energy on the reviews.